pkg/agave/assets/update.sh

```diff
@@ -0,0 +1,11 @@
+# -*- mode: shell-script -*-
+# shellcheck shell=bash
+
+step::00::update-run-validator() {
```

Comment:
We need this to be more than just this script right?  Wouldn't updating also include potentially updating software?  The other scripts should be idempotent, so "updating" should just consist of running the scripts.

Reply:
Ok so use install script for create, update in pulumi? I had that thought as well and will switch to that so just closing this PR and we can use install as is just use the pulumi change to run update.

https://github.com/abklabs/pulumi-svmkit/pull/20